### PR TITLE
Add `global_` parameter to `Channel.set_qos` method

### LINKED ISSUE #169: all_channels at the RobustChannel's QoS
What's the reason of unavailability of `all_channels` parameter at the `RobustChannel.set_qos` method?

https://github.com/mosquito/aio-pika/blob/57af0c3f3f81f49e9593f7d96ef624582ce3be79/aio_pika/robust_channel.py#L87-L90

### PR DESCRIPTION
**Pull request by decaz**
Closes #169.

### CODE REVIEW

**Comment by mosquito** [comment]
passing kwargs will be better and backward compatible.

**Comment by decaz** [comment]
What kind of backward compatibility do you mean?

If you mean remove of `all_channels` parameter then I think nobody used it because it wasn't used inside of library in any way and didn't gave any effect. Can we can just remove it to not to bring some complexity in such a simple case?

If you mean incompatibility with `aiormq` then we can change minimal required version of it.

What do you think?

**Comment by mosquito** [comment]
1. Yes! I want to restrict `aiormq>3,<4`.
2. `all_channels` might be used by 4.x.x version users.

**Comment by mosquito** [comment]
`all_channels` actually means `global_` in this case.
I see two ways:
1. marking `all_channels` deprecated and add `global_` as an alias for `all_channels`.
2. just add alias for `all_channels` to `global_`

**Comment by decaz** [comment]
Why do you want to save parameter that was used by **previous major** version? There were another breaking changes at 5 version (for example https://github.com/mosquito/aio-pika/issues/200#issuecomment-467613597). And now `all_channels` has no effect at all and if someone is using it within 5th version then he will have to just remove it and that's all.

File: aio_pika/channel.py [hunk 1]
```diff
@@ -263,13 +263,14 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
     async def set_qos(
         self, prefetch_count: int = 0, prefetch_size: int = 0,
-        all_channels: bool = False, timeout: TimeoutType = None
```
[the remark this answers is not in the document]
Comment (decaz) [reply aio_pika/channel.py]: @mosquito fixed.

File: setup.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
     packages=find_packages(exclude=['tests']),
     package_data={'aio_pika': ['py.typed']},
     install_requires=[
-        'aiormq>=2.9,<3',
+        'aiormq>=3,<4',
```
Comment (mosquito) [inline setup.py]: please add `>=3.0.1,<4`

Comment (mosquito) [reply setup.py]: Or just rebase against master

**Comment by decaz** [comment]
@mosquito done.

**Comment by mosquito** [comment]
Clean work. But I'm not sure about `RobustConnection` behaviour. Might be better to ignore it for `RobustChannel`?

**Comment by decaz** [comment]
Why do we need to change behavior if connection is robust? If it's robust then how can channel QoS be global among multiple consumers?

**Comment by mosquito** [comment]
After reconnection new connection has no had globally set QoS. And connection state will be different after reconnecting. The correct way is propagating last set QoS with the `global_` parameter into `RobustConnection` from `RobustChannel` as a connection property.

**Comment by mosquito** [comment]
Please rebase against the master and I hope it will be merged soon.

**Comment by decaz** [comment]
@mosquito rebase done.

**Pull request merged**